code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -502,6 +502,19 @@
 	M.adjustToxLoss(1)
 	..()
 
+/datum/reagent/somnistimine
+	name = "Somnistimine"
+	id = "somnistimine"
+	description = "A powerful tranquilizer."
+	reagent_state = LIQUID
+	color = "#2E2E61"
+	metabolization_rate = 0.4
+
+/datum/reagent/somnistimine/on_mob_life(mob/living/M)
+	if(current_cycle>= 3)
```

Comment:
I moved away from the CTF3 spray. Just takes too much time to use. Leaves too much room for error unless you got fireproof clothes. Its effective, but it also can be your own grave... especially if yer a drask. 

Bag of holding is still limited, but yes, it can fit 3-5 times the stuff normal backpack can. Also it can fit stuff labeled as "large" (ion rifle, other backpacks etc.) Also outside science it can be a pain to obtain unless scientists are generous. Lately i have seen that miners cannot really do much and resources for it are missing.

I remember there was a huge debate about spliting up science and merging RnD and robotics as a one job. Even maps were made where the two different jobs had their own "wings" in science. No idea why it never got thru. Was around last years spring or something it was debated on.